src/sentry/static/sentry/app/views/projectInstall/issueAlertOptions.tsx

```diff
@@ -144,7 +139,6 @@ class IssueAlertOptions extends AsyncComponent<Props, State> {
             min="0"
             name=""
             value={this.state.threshold}
-            placeholder={this.state.placeholder}
```

Comment:
Why was placeholder previously in state? Where was it coming from - we should probably still have fixed value for the placeholder (i.e. if they delete the default value, we should still show something)

Reply:
Initially the placeholder value is fetched from the end point. Added the default value as a placeholder value.